### PR TITLE
Refactoring: FStep -> IntermediateStep and related functions.

### PR DESCRIPTION
**Pull request by ryukzak**

### CODE REVIEW

File: src/NITTA/Utils.hs [hunk 1]
```diff
@@ -122,15 +122,12 @@ endpointAt t p =
...
-getFB Step{pDesc} | FStep fb <- descent pDesc = Just fb
-getFB _ = Nothing
+getIntermediate Step{pDesc} | IntermediateStep f <- descent pDesc = Just f
+getIntermediate _ = Nothing
 
-getFBs p = mapMaybe getFB $ sortOn stepStart $ steps p
-
-getFunction Step{pDesc} | FStep role <- descent pDesc = Just role
-getFunction _ = Nothing
+getIntermediates p = mapMaybe getIntermediate $ sortOn stepStart $ steps p
```
Comment (ArturGogiyan) [inline src/NITTA/Utils.hs:130]: as I see, it is just renaming?

Comment (ArturGogiyan) [inline src/NITTA/Utils.hs:130]: as I see, it is just renaming?

Comment (ryukzak) [reply src/NITTA/Utils.hs:130]: Yes

Comment (ryukzak) [reply src/NITTA/Utils.hs:130]: Maybe, you know any other places, that are forgotten by me.

**Pull request merged**